driver/utils/type_info.h

```diff
@@ -638,8 +638,17 @@ namespace value_manip {
     struct from_value<std::string>::to_value<std::string> {
         using DestinationType = std::string;
 
-        static inline void convert(const SourceType & src, DestinationType & dest) {
-            dest = src;
+//        Prevent copying std::string
+//        static inline void convert(const SourceType & src, DestinationType & dest) {
+//            dest = src;
+//        }
+
+        static inline void convert(SourceType && src, DestinationType & dest) {
+            dest = std::move(src);
+        }
+
+        static inline void convert(SourceType & src, DestinationType & dest) {
+            dest = std::move(src);
```

Comment:
This one is very dangerous thing to do, in terms of behavior/style. If argument is meant to be consumed - it should be rvalue reference. If it is meant to be modified (in non-destructive way) then it should be reference. Const reference otherwise.

Reply:
Ok, np we can remove this overload and `std::move()` all arguments that are safe to steal from on caller side.